pkg/controller/config/config_controller.go

```diff
@@ -76,9 +84,24 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 		secClient: secClient,
 		pipeline:  pipeline,
 		addons:    addons,
+		community: community,
 	}, nil
 }
 
+func fetchCommuntiyResources(mgr manager.Manager) (mf.Manifest, error) {
```

Comment:
Would rather like to write `func (mgr *manager.Manager) fetchCommuntiyResources(mf.Manifest, error)`?

Reply:
done